media/website/content/_index.md

```diff
@@ -663,31 +665,50 @@ group_mapping:
 The records considered in the computation can be limited to all records which finished in the past X seconds via the `duration` field (in seconds).
 Omitting this field takes all records in the database into account.
+The frequency of recalculating the priorities can be set via the `frequency` field.
 Via the `group_mapping` field, it is possible to attach certain additional information to the individual groups which are to be considered in the calculation.
 In the example configuration above are three groups `group{1,2,3}`, where each has a corresponding partition `part{1,2.3}`.
-These mappings can be accessed when constructing the `command`s which will be executed after computing the priorities by using `{N}` where `N` corresponds to number of the element in the list of the `group_mapping`.
+These mappings can be accessed when constructing the `commands` which will be executed after computing the priorities by using `{N}` where `N` corresponds to number of the element in the list of the `group_mapping`.
 For instance, for `group1`, the string `{1}` indicates `part1` while for `group2` the same string `{1}` indicates `part2`.
-The `command` field in the configuration illustrates the usage of these mappings.
-This allows one to adapt the command for the various groups involved.
-In the `command` field one can also see a string `{priority}`, which will be replaced by the computed priority for the group.
+The group name can be accessed via the `{group}` string.
+The `commands` field in the configuration illustrates the usage of these mappings.
+This allows one to adapt the commands for the various groups involved.
+In the `commands` field one can also see a string `{priority}`, which will be replaced by the computed priority for the group.
 Another special string, `{resources}` is available, which is replaced by the computed provided resource per group.
-The command is executed for each group separately and multiple commands can be provided.
+The command is executed for each group separately and multiple commands can be provided with a list.
 The verbosity of logging can be set with the `log_level` option. Possible values are `trace`, `debug`, `info` (default), `warn`, and `error`.
+The priority plugin allows for real-time monitoring of the computed resources and priorities via a prometheus endpoint.
+Per default, the prometheus endpoint is disabled.
+It can be enabled by adding the `prometheus` block to the configuration or by setting the `enable` field of this block to `true`.
+Inside the `prometheus` block, the address and port of the HTTP server that provides the prometheus metrics can be specified via the `addr` and `port` fields.
+The metrics will then be available at `addr>:<port>/metrics`
+The `metrics` list specifies the metrics that are exported.
+Right now the values `ResourceUsage` (for the amount of provided resources in the given duration)
```

Comment:
Is there something missing in this sentence?

Reply:
Ah right, I forgot to copy the second line from the migration guide ...
Fixed